interfaces/builtin/u2f_devices.go

```diff
@@ -188,9 +188,9 @@ var u2fDevices = []u2fDevice{
 		ProductIDPattern: "8055",
 	},
 	{
-		Name:             "TrustKey TrustKey G310H",
+		Name:             "TrustKeys FIDO2 U2F",
 		VendorIDPattern:  "311f",
-		ProductIDPattern: "4a2a",
+		ProductIDPattern: "4a2a|a6e9",
```

Comment:
Where can I lookup the `ProductIDPattern` for the TrustKey devices? It would be nice to have a reference similar to the link at the top if that exists.

Reply:
I am not affiliated with TrustKey, so I don't have this information either.

I have obtained the additional pattern from a TrustKey device that I own.

Reply:
I found an updated list from `libfido2` https://github.com/Yubico/libfido2/blob/main/udev/70-u2f.rules and indeed `a6e9` belongs to TrustKey, it would be great if you could add the link above the list.

```
# TrustKey Solutions FIDO2 T120 by eWBM Co., Ltd.
KERNEL=="hidraw*", SUBSYSTEM=="hidraw", ATTRS{idVendor}=="311f", ATTRS{idProduct}=="a6e9", TAG+="uaccess", GROUP="plugdev", MODE="0660"
```